vtexdocs/helpcenter
# Feat/announcement

#### What is the purpose of this pull request?

Adicionar a página individual para Comunicados.

Mais informações [aqui](https://www.notion.so/vtexhandbook/P-ginas-comunicados-f5639e8cbc0d41c68185e69eb054eb1b?pvs=4#a683557a6c6b45b0aed48afc4b3f1256)

#### What problem is this solving?

Adição das novas funcionalidades propostas.

#### How should this be manually tested?

Página de listagem de announcements:
1. Acesse https://deploy-preview-28--leafy-mooncake-7c2e5e.netlify.app/pt/announcements
2. Use as funcionalidades da página, busca, ordenação e paginação e confira se estão funcionando como o esperado.
3. Abra um dos cards de comunicado e verifique se entrou na documentação certa.

Página de artigo announcement:

Para testar todas as adições será necessário utilizar o modo de preview.

1. Acesse https://deploy-preview-28--leafy-mooncake-7c2e5e.netlify.app/api/preview?branch=test/see-also
4. No topo da página deve aparecer "🚧 You are currently using branch in preview mode. This content may differ from the published version."
5. Acesse alguma página de comunicado e compare com o protótipo.
Não será possível utilizar a sidebar para navegar porque uma mudança foi feita no navigation, então, siga esses passos: acesse o repositório help-center-content e a [pasta de announcements](https://github.com/vtexdocs/help-center-content/tree/main/docs/announcements), entre na pasta da linguagem atual, copie o nome de um dos arquivos e cole no fim da seguinte URL: https://deploy-preview-28--leafy-mooncake-7c2e5e.netlify.app/pt/announcements/. Obtendo por exemplo: https://deploy-preview-28--leafy-mooncake-7c2e5e.netlify.app/pt/announcements/3ds2-authentication-flow-accept-online-payments-more-securely-1. Agora você deve visualizar uma página de documentação.
6. Para visualizar a seção de "Acesse mais comunicados", é necessário acessar um documento em específico (que contém os dados para essa seção): https://deploy-preview-28--leafy-mooncake-7c2e5e.netlify.app/pt/announcements/manage-your-fbaclassic-dba-orders-in-one-place


#### Screenshots or example usage

![Captura da Web_5-2-2024_201312_localhost](https://github.com/vtexdocs/helpcenter/assets/62757720/07d4b786-b604-4ac7-a0cb-a6623bf1f4d3)

![image](https://github.com/vtexdocs/helpcenter/assets/62757720/6fc4b6c3-a45f-49b2-aff5-22608881a336)

#### Types of changes
